doc/common/README.md

```diff
@@ -22,10 +22,6 @@
 
 Cloud iNfrastructure Telco Taskforce [Introduction](chapter00.md). 
 
-<a name="4.0"></a>
-## Approach
-Different specification types within CNTT are related to each other and requires alignments between different parts. The Approach CNTT is taking to tackle those issues is described in [here](./approach.md).
-
 <a name="5.0"></a>
 ## Use Cases
 CNTT Addresses wide range of use cases from Core all the way to the Edge of the Network. Different use cases supported by CNTT specifications are described in [here](./usecases.md).
```

Comment:
nits
```suggestion
The CNTT specifications address a wide range of use cases from the Core all the way to the Edge of the Network. The different use cases supported by the CNTT specifications are described [here](./usecases.md).
```

Reply:
Maybe in addition replace CNTT by Anuket.

Reply:
I would suggest to that once Anuket is announced and at the beginning of a new release cycle.